pkg/services/sqlstore/migrator/postgres_dialect.go

```diff
@@ -140,6 +140,37 @@ func (db *Postgres) CleanDB() error {
 	return nil
 }
 
+func (db *Postgres) TruncateDB() error {
+	sess := db.engine.NewSession()
+	defer sess.Close()
+
+	for _, table := range db.engine.Tables {
+		switch table.Name {
+		case "dashboard_acl":
+			// keep default dashboard permissions
+			if _, err := sess.Exec(fmt.Sprintf("DELETE FROM %v WHERE dashboard_id != -1 AND org_id != -1;", db.Quote(table.Name))); err != nil {
+				return errutil.Wrapf(err, "failed to truncate table %q", table.Name)
+			}
+			if _, err := sess.Exec(fmt.Sprintf("ALTER SEQUENCE %v RESTART WITH 3;", db.Quote(fmt.Sprintf("%v_id_seq", table.Name)))); err != nil {
+				return errutil.Wrapf(err, "failed to reset table %q", table.Name)
+			}
+		default:
+			if table.Name == "" {
```

Comment:
I think this should be in case statement above, `case "":`, instead of nested in the default?

Reply:
Good point